tools/wheel_resolver/test/BUILD

```diff
@@ -1,5 +1,17 @@
 subinclude("//build_defs:python")
 
+package(
+    python = {
+        "WHEEL_REPO": "https://get.please.build/third_party/python/py3",
+        "WHEEL_NAME_SCHEME": [
+            "{url_base}/{package_name}-{version}-${{OS}}_${{ARCH}}.whl",
+            "{url_base}/{package_name}-{version}.whl",
+            "https://files.pythonhosted.org/packages/py3/{initial}/{package_name}/{package_name}-{version}-py3-none-any.whl",
+            "https://files.pythonhosted.org/packages/py2.py3/{initial}/{package_name}/{package_name}-{version}-py2.py3-none-any.whl",
+        ],
+    },
+)
+
```

Comment:
I copied this over from `third_party/python/BUILD` as there's a `python_wheel` rule below that needs it. Not sure if this change is the correct one.

Reply:
@samwestmoreland can you confirm?

Reply:
Yeah as we were saying on Friday it shouldn't need it because the python_wheel() should fall back onto the wheel resolver tool. The issue was to do with the wheel resolver tool not being able to find old protobuf versions (re-opened https://github.com/please-build/python-rules/issues/19).